{
  "metadata": {
    "timestamp": "2025-02-24T00:51:14Z"
  },
  "data": [
    {
      "cve": {
        "id": "CVE-2018-10878",
        "description": "A flaw was found in the Linux kernel's ext4 filesystem. A local user can cause an out-of-bounds write and a denial of service or unspecified other impact is possible by mounting and operating a crafted ext4 filesystem image.",
        "metadata": {
          "published_date": "2018-07-26T18:29:00.307",
          "last_modified_date": "2024-11-21T03:42:12.333",
          "confidence_level": "medium",
          "severity": "high"
        },
        "impact": {
          "cisa_kev": false,
          "reported_exploited": false,
          "exploit_maturity": "poc",
          "automatable": false
        },
        "counts": {
          "public_exploit_count": 2
        },
        "epss": {
          "score": 0.00044,
          "percentile": 0.12419
        },
        "metrics": [
          {
            "source": "nvd@nist.gov",
            "type": "Primary",
            "cvss_version": "3.1",
            "vector_string": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H",
            "attack_vector": "LOCAL",
            "base_score": 7.80000019073486
          },
          {
            "source": "secalert@redhat.com",
            "type": "Secondary",
            "cvss_version": "3.0",
            "vector_string": "CVSS:3.0/AV:P/AC:H/PR:N/UI:N/S:U/C:N/I:L/A:H",
            "attack_vector": "PHYSICAL",
            "base_score": 4.80000019073486
          },
          {
            "source": "nvd@nist.gov",
            "type": "Primary",
            "cvss_version": "2.0",
            "vector_string": "AV:L/AC:L/Au:N/C:P/I:P/A:C",
            "attack_vector": "LOCAL",
            "base_score": 6.09999990463257
          }
        ],
        "timeline": {
          "references": [
            {
              "type": "reference",
              "url": "https://bugzilla.kernel.org/show_bug.cgi?id=199865",
              "source_type": "poc",
              "source": "nvd",
              "description": "",
              "published": "2018-07-26T05:00:00.000Z"
            },
            {
              "type": "reference",
              "url": "https://bugzilla.kernel.org/show_bug.cgi?id=199865",
              "source_type": "poc",
              "source": "nvd",
              "description": "",
              "published": "2018-07-26T05:00:00.000Z"
            }
          ],
          "repositories": []
        }
      }
    }
  ]
}